Kudu.Services.Web/_Layout.cshtml

```diff
@@ -36,7 +36,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="~/"><img src="/Content/Images/kudu.svg" height="22" width="22">Kudu</a>
+                <a class="navbar-brand" href="~/"><img src="/Content/Images/kudu.svg" height="22" width="22" style="margin-right: -3px">udu</a>
```

Comment:
Yes! sorry the commit was already in my repo when I created the branch. I'll make sure to not merge it.